assets/js/search-utilities.js

```diff
@@ -187,31 +188,63 @@ function constructTitleSuggestions(entries, resultTitlesIds, suggestionDescripti
   }
 }
 
+function constructTitle(storedEntry) {
+  const title = document.createElement('div');
+  title.textContent = storedEntry.title;
+  title.classList.add('suggestion__title');
+
+  return title;
+}
+
+function constructAnchor(title, storedEntry) {
+  const a = document.createElement('a');
+  a.href = storedEntry.href;
+  a.appendChild(title);
+
+  return a;
+}
+
+function constructFullSearchLengthInfo(allMatches) {
+  const fullResultsLengthInfo = document.createElement('span');
+  fullResultsLengthInfo.style['font-size'] = '13px';
+  fullResultsLengthInfo.style.display = 'inline';
+  fullResultsLengthInfo.innerHTML = `[${allMatches.length} more occurrences]`;
```

Comment:
Maybe total occurrences because here we list all the matches count but we have displayed some of them in the short description preview.
Or fix it to be `allMatches.length - ${showedMatches}`

Reply:
Or leave it like it is now it is not a big deal :D

Reply:
Fixed.